RELEASE.md

```diff
@@ -1,33 +1,47 @@
-## Making a release of the Checker Framework Gradle plugin
+## Making and merging pull requests
 
-This document describes how to make a release of the Checker Framework
-Gradle plugin, which this repository contains.
+The Checker Framework Gradle plugin (which this repository contains)
+uses a continuous-delivery style of releases: any time that the
+behavior of the plugin changes, a new release is cut.
 
-### Prequisites
+### Making a pull request
+
+When you make a pull request, update the version string on your branch
+using these steps:
+0. Make the changes you wish to merge into the plugin.
+1. Choose a new version string. Please try to respect
+[semantic versioning](https://semver.org/). Do not augment the major
+version without explicit approval from all the maintainers.
+2. Change the version of the plugin. Search for the current plugin version
```

Comment:
Both 1 and 2 are part of "change the version", so this is confusing.  I would rewrite with one top-level item and two subitems.

Reply:
All of these steps are part of "change the version". These are two distinct steps that require the reader to think about and do different kinds of things, so I think they should be separate.

I would be willing to compress them into a single step if we also compress the remainder of these instructions into a single step: "change the version number, respecting semantic versioning". This is much more succinct, but leaves a lot for the reader to figure out. If we're going to be explicit about each step, we should be explicit about each step.